scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/tasks/TasksListGrid.java

```diff
@@ -182,17 +183,19 @@ public void tasksUpdated(List<Task> tasks, long totalTasks) {
             }
         }
 
-        this.ds.setTestData(data.toArray());
+        this.ds.setCacheData(data.toArray());
         data.destroy();
         applyCurrentLocalFilter();
     }
 
     @Override
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)